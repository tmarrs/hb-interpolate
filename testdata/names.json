{
  "names": [{
    "first": "Ezekiel",
    "last": "Riker"
  }, {
    "first": "Klem",
    "last": "Fitzsimmons"
  }, {
    "first": "Larson",
    "last": "Picard"
  }]
}
