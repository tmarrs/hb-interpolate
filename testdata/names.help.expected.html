<!DOCTYPE html>
<html>
  <head>
    <title>Names</title>
  </head>
  <body>
    <ul>
      <li>Mr. Ezekiel Riker</li>
      <li>Mr. Klem Fitzsimmons</li>
      <li>Mr. Larson Picard</li>
    </ul>
  </body>
</html>
